tests/taxii2/test_taxii2_sqldb.py

```diff
@@ -921,9 +944,7 @@ def test_add_objects(
         )
         assert db_job_detail.job_id == db_job.id
         assert db_job_detail.stix_id == obj["id"]
-        assert db_job_detail.version == datetime.datetime.strptime(
-            obj["modified"], DATETIMEFORMAT
-        ).replace(tzinfo=datetime.timezone.utc)
+        assert db_job_detail.version == get_object_version(obj)
```

Comment:
Let's have a unit test for `get_object_version` specifically. This integration testing isn't going to catch changes in the function (e.g. swap order for `created` and `modified` and similar).